parser/v2/scriptparser.go

```diff
@@ -58,6 +59,21 @@ func (p scriptElementParser) Parse(pi *parse.Input) (n Node, ok bool, err error)
 		return
 	}
 
+	// Check for a type attribute. If it is not text/javascript, we need to parse the contents as raw text.
```

Comment:
Is this comment correct?

The behaviour should be that "If there's a type attribute _and_ it's not text/javascript, then we need to parse the contents as raw text.", since, without a type attribute, the default is JavaScript.